### ISSUE TITLE
PROBLEM

### ISSUE THREAD

**Issue by gonzo91** [finndev/PokeBuddy#1056]
after a few use pokebuddy my screen becomes so, my character does not move anymore, and above all I do not see the pokestop! some idea?
![poke](https://cloud.githubusercontent.com/assets/21011664/17644759/c57fb59a-618f-11e6-99f6-7eb6ba3eddfc.jpeg)

**Comment by iPetrov2** [comment]
Thats a Soft Ban. wait for about a Day. then try again. and if it works. good. when your done using it. wait about 12-24 hours before logging in on Pokemon Go

**Comment by AlexCtasy** [comment]
I dunno, why ppl still boting while niantic are banning a lot of ppl for use bots,

**Comment by iPetrov2** [comment]
@AlexCtasy if people want to use the bot let them use the bot. the only bans ive been getting have been soft bans cause i dont wait long enough to log into my pokemon go account

**Comment by gonzo91** [comment]
thanks @iPetrov2 ! i will follow your advice! 
@AlexCtasy i have 4-5 secondary accounts like that one above. I keep try to use the bot... if they ban me I will not kill myself for sure and then i keep the others account

**Comment by iPetrov2** [comment]
@gonzo91 Another Tip. If your using a Main Account i wouldnt Use a Bot on a Main account just in case the rest of your accounts get banned. lol. im only botting on 1 account and i havent gotten banned. Well Besides the Soft Bans. And Try VPN if you arent using it already.

**Comment by NegroSorete** [comment]
I think you get IP Ban, restart your modem or router

**Comment by gonzo91** [comment]
yes yes i'm using not my main account . i'm doing as you do ( only secondary accounts ) .
Thanks again

**Comment by gonzo91** [comment]
@negroSorete
Later i try and i let you know

**Comment by NegroSorete** [comment]
You have dynamic ip right?

**Comment by gonzo91** [comment]
mmm i think so but i'm not 100% sure... i'm a disaster with pc

**Comment by gonzo91** [comment]
@negroSorete yes, was a ip ban ... now it works

**Comment by engemerson** [comment]
i have IP static, qhats i do?

**Comment by NegroSorete** [comment]
@engemerson you have to change the MAC Address

**Comment by engemerson** [comment]
Oh Good, vur what program o can use for tarde my MAC?
